client_app/src/VolunteerDashboard.js

```diff
@@ -0,0 +1,59 @@
+function VolunteerDashboard() {
+  return (
+    <div class="volunteer-dashboard">
+      <div class="column column-1">
+        <h3>Currently Registered</h3>
+        <div class="card">
+          <h4>St Patrick Center Shelter for Women</h4>
+          <p>St. Louis MO, 63101</p>
+          <p>THU,FRI - 10:00 AM to 11:00 AM</p>
+        </div>
+        <div class="card">
+          <h4>Covenant House Missouri</h4>
+          <p>2727 North Kingshighway Boulevard • St. Louis MO, 63113</p>
+          <p>SAT - 01:00 PM to 2:00 PM</p>
+        </div>
+        <div class="card">
+          <h4>The Haven of Grace</h4>
+          <p>1225 Warren Street • St. Louis MO</p>
+          <p>SUN - 4:00 PM to 1:00 PM</p>
+        </div>
+      </div>
+
+      <div class="column column-2">
+        <h3>Shelters looking for Volunteers</h3>
```

Comment:
It would be nice if you could add the dynamic search for shelters looking for volunteers (what you currently have under shelters) here. Maybe limit it to 30 miles and display 3 shelters with the biggest need for volunteers?  Then add a button at the bottom leading the user to search for more shelter sign up options (what's currently under Shelters).